sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -159,7 +159,7 @@ The following section provides several code snippets using the `client` [created
 ### Async examples
 * [Detect Language Asynchronously](#detect-language-asynchronously)
 * [Recognize Entities Asyncronously](#recognize-entities-asynchronously)
-* [Recognize Healthcare Entities Asyncronously](#recognize-healthcare-entities-asynchronously)
+* [Analyze Healthcare Entities Asyncronously](#analyze-healthcare-entities-asynchronously)
```

Comment:
nit: spelling here and for the above line
```suggestion
* [Analyze Healthcare Entities Asynchronously](#analyze-healthcare-entities-asynchronously)
```

Reply:
thank you!